src/components/dropdown/dropdown.js

```diff
@@ -128,7 +128,7 @@ class DropDown extends Component {
    * @memberof DropDown
    */
   renderNativeSelect(): React$Element<any> {
-    return <Menu options={this.props.options} onMenuChosen={o => this.onMenuChosen(o)} onClose={() => this.onClose()} />;
+    return <Menu pushRef={this.props.pushRef} options={this.props.options} onMenuChosen={o => this.onMenuChosen(o)} onClose={() => this.onClose()} />;
```

Comment:
check in cvaa portal mode (pushref is not passed there)

Reply:
checked - it is ok. in Portal mode the ref comes from the select inside menu, in regular mode the ref comes from the wrapping div in dropdown